CAP/gap/PrintingFunctions.gi

```diff
@@ -1,8 +1,8 @@
-#############################################################################
-##
-##                                               CAP package
-##
-##  Copyright 2016, Sebastian Gutsche, TU Kaiserslautern
+#
+# CAP: Categories, Algorithms, Programming
+#
+# Implementations
+#
 ##                  Sebastian Posur,   RWTH Aachen
```

Comment:
Why did my name survive the cleanup here (and in the instances below)? ;-)

Reply:
Thanks for spotting this! I indeed seem to have stopped at some point in the alphabet and missed all files after this one, not sure why :D Should be fixed :-)